test_conformance/api/test_queue_properties_queries.cpp

```diff
@@ -267,3 +267,79 @@ int test_queue_properties_queries(cl_device_id deviceID, cl_context context,
     }
     return error;
 }
+
+int test_set_command_queue_property(cl_device_id deviceID, cl_context context,
+                                    cl_command_queue queue, int num_elements)
+{
+    int err;
+
+    // Required minimum command queue properties
+    std::vector<cl_command_queue_properties> queue_property_options = {
+        0, CL_QUEUE_PROFILING_ENABLE
```

Comment:
Why such initialization. What is purpose ? It can brake `queue_property_options` setting later (lines 289 and 290).